js/NarrativeView.js

```diff
@@ -25,6 +25,8 @@ class NarrativeView extends ComponentView {
 
     this.model.set('_isInitial', true);
     this.model.set('_activeItemIndex', 0);
+    this.model.set('_isInview', false);
+    this.model.set('_isFullyLoaded', false);
```

Comment:
```suggestion
    this_isFullyLoaded = false;
```
Would you mind changing both of these view states into view properties rather than model attributes? Then I can have a better look Tuesday/Wednesday if that's ok?